core/src/consensus/solo/mod.rs

```diff
@@ -120,6 +120,8 @@ impl ConsensusEngine for Solo<CodeChainMachine> {
         for (address, reward) in rewards {
             self.machine.add_balance(block, &address, reward)?;
         }
+
+        stake::on_term_close(block.state_mut())?;
```

Comment:
`on_term_close` is always paired with `self.machine.increase_term_id()`. Why don't we increase term id in the `on_term_close` ?

Reply:
Looks good. I'll fix it.